app/src/main/java/org/wikipedia/activity/BaseActivity.kt

```diff
@@ -97,7 +97,7 @@ abstract class BaseActivity : AppCompatActivity(), ConnectionStateMonitor.Callba
         }
 
         // Conditionally execute all recurring tasks
-        RecurringTasksExecutor(WikipediaApp.instance).run()
+        RecurringTasksExecutor(lifecycleScope).run()
```

Comment:
Actually I don't think the Recurring Tasks should be bound to the `lifecycleScope` of this activity (they should be allowed to run independently of any activity). Can we make them use `MainScope` instead?

Reply:
I see. Done.